test/README.md

```diff
@@ -1 +1,12 @@
-See https://github.com/WebAssembly/wasi-testsuite
+# Running tests
+The following command executes wasi-threads tests using wasmtime runtime.
+
+```bash
+git clone -b prod/testsuite-all https://github.com/WebAssembly/wasi-testsuite
+cd wasi-testsuite/
+TEST_RUNTIME_EXE="wasmtime <TBD flags>" python3 test-runner/wasi_test_runner.py \
```

Comment:
Please note `TEST_RUNTIME_EXEC` is a feature in PR right now: https://github.com/WebAssembly/wasi-testsuite/pull/46